client/lib/plans/features-list.js

```diff
@@ -570,7 +570,7 @@ export const FEATURES_LIST = {
 	[ constants.FEATURE_13GB_STORAGE ]: {
 		getSlug: () => constants.FEATURE_13GB_STORAGE,
 		getTitle: () =>
-			i18n.translate( '{{strong}}13GB{{/strong}} Storage Space', {
+			i18n.translate( '{{strong}}13 GB{{/strong}} Storage Space', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **25** times:
`translate( '{{strong}}200 GB{{/strong}} Storage Space' )` *ES Score: 8*